dependencies/adi/command_parser/CMakeLists.txt

```diff
@@ -5,6 +5,19 @@ add_library(${PROJECT_NAME} STATIC
     command_parser.cpp
 )
 
+if(USE_GLOG)
```

Comment:
I would not make command_parser depend of glog. You should stick to have the parser return error codes and let client code to print error messages or help menu.

Reply:
Parser is no loger depended in glog. Kept the LOGs commented to further help me until they are no longer needed